pom.xml

```diff
@@ -176,44 +127,10 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <groupId>org.mortbay.jetty</groupId>
-      <artifactId>jetty</artifactId>
-      <version>${jetty.version}</version>
-      <scope>runtime</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.mortbay.jetty</groupId>
-      <artifactId>jetty-util</artifactId>
-      <version>${jetty.version}</version>
-      <scope>runtime</scope>
-    </dependency>
-    <dependency>
-      <groupId>javax.servlet</groupId>
-      <artifactId>servlet-api</artifactId>
-      <version>2.5</version>
-      <scope>runtime</scope>
-    </dependency>
-    <dependency>
-      <groupId>com.github.samtools</groupId>
-      <artifactId>htsjdk</artifactId>
-      <version>1.128</version>
-      <exclusions>
-        <exclusion>
-          <groupId>org.testng</groupId>
-            <artifactId>testng</artifactId>
-        </exclusion>
-      </exclusions>
-    </dependency>
-    <dependency>
       <groupId>com.google.cloud.genomics</groupId>
       <artifactId>gatk-tools-java</artifactId>
       <version>1.0</version>
```

Comment:
Looks like the latest version is `1.1` based on this:

https://oss.sonatype.org/#nexus-search;quick~gatk-tools-java